osg-token-renewer.py

```diff
@@ -110,6 +110,12 @@ def mktoken(cfg):
     prog    = ['oidc-token']
     aud     = option_if('aud',   cfg.get("audience")    )
     scope   = option_if('scope', cfg.get("scope")       )
+    if scope == []:
+        # A blank scope tells oidc-token to request the default scopes of
```

Comment:
```suggestion
        # FIXME: A blank scope tells oidc-token to request the default scopes of
```

Just so it's easier to find later

Reply:
Dave and I discussed this off-PR: this is currently required for CILogon (but won't be after April) but this is generally good to have as a backstop so we're keeping it permanently